packages/samples/atomic-react/public/index.html

```diff
@@ -10,5 +10,5 @@
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
   </body>
-  <script src="dist/app.js"></script>
+  <script src="dist/App.js"></script>
```

Comment:
@olamothe should it be app or App? it seems to work like this, but I haven't changed anything so I'm confused